maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```
@@ -296,6 +331,33 @@ private boolean connectMySqlDatabase(Server server) throws MojoExecutionExceptio
         }
     }
 
+    private List<String> listTables(Server server) throws MojoExecutionException {
+        String uri = server.getDbUri();
+        uri = uri.substring(0, uri.lastIndexOf("/"));
+        DBConnector connector = null;
+        final int tableNameColumnIndex = 3;
+        try {
+            connector = new DBConnector(uri, server.getDbUser(), server.getDbPassword(), server.getDbName());
+            DatabaseMetaData md = connector.getConnection().getMetaData();
+            ResultSet rs = md.getTables(server.getDbName(), null, null,new String[] {"TABLE"});
+            List<String> tableList = new ArrayList<>();
+            while (rs.next()) {
+                tableList.add(rs.getString(tableNameColumnIndex));
+            }
+            return tableList;
+        } catch (SQLException e) {
+            return null;
```

Why would you ever ignore an exception and just return null?